### PR TITLE
Spelling_and_Grammatical_error_fixed

### PR DESCRIPTION
**Pull request by Abubakrce19**
This includes a minor but Impactful Spelling correction and a few Grammatical fixes. Thank you

### CODE REVIEW

File: docs/docs/Enterprise/audit_logs.md [hunk 1]
```diff
@@ -39,17 +39,17 @@ The dropdown will list all the apps present in your account. Choose an app to fi
 | USER_INVITE | You can invite users to your account from `Manage Users` section and an event is audited everytime an invite is sent. |
 | USER_INVITE_REDEEM | This event is recorded whenever an invite is redeemed. |
 | APP_CREATE | This event is recorded when a user creates a new app. |
-| APP_UPDATE | This event is recorded whenever actions like renaming the app, making the app public, editing shareable link, or deploying the app are made. |
-| APP_VIEW | This event is logged when someone views the launched app. (public apps isn't accounted for) |
+| APP_UPDATE | This event is recorded whenever actions like renaming the app, making the app public, editing shareable links, or deploying the app are made. |
```
Comment (withshubh) [inline docs/docs/Enterprise/audit_logs.md:42]: Please revert this.

**Comment by Abubakrce19** [comment]
Ok

File: docs/docs/Enterprise/audit_logs.md (hunk 1 above)
Comment (Abubakrce19) [reply docs/docs/Enterprise/audit_logs.md:42]: Revert change on line no 42 and 43 right?
Thank you

Comment (withshubh) [reply docs/docs/Enterprise/audit_logs.md:42]: Just the 42 one.

Comment (Abubakrce19) [reply docs/docs/Enterprise/audit_logs.md]: Done!

**Review by withshubh — APPROVED** [review]

File: docs/docs/Enterprise/audit_logs.md (hunk 1 above)
Comment (AbubakrChan) [reply docs/docs/Enterprise/audit_logs.md]: @withshubh Thank you! for approving these changes.
I have a questions.
This PR is not merged, will this still be counted in Hacktoberfest?

**Pull request merged**